python/dataproc_templates/bigquery/README.md

```diff
@@ -0,0 +1,20 @@
+## BigQuery To GCS 
+
+General Execution:
+
+```
+export GCP_PROJECT=<project_id>
+export SUBNET=<region>
+export JARS="gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
```

Comment:
Are we sure we need this jar with python ?

Reply:
yes, pyspark requires connector to be manually specified

Reply:
To elaborate here a bit, the Java implementation ships with many JARs when it's packaged into a JAR itself. For example, the Spark BigQuery connector is shipped right with the templates instead of being specified at job submission time.

For Python, shipping JARs with the code it not an option, they need to be passed at job submission.

We could document clearly the required JARs for each template (and maybe provide recommended URIs to them if available).